Documentation/Signoffs/Power System

```diff
@@ -0,0 +1,33 @@
+## Required Format and Sections of the signoff request
```

Comment:
This is not a markdown file. Change the file extension to be .md

Reply:
I changed the filetype as requested